man/machinery-inspect-docker.1.md

```diff
@@ -0,0 +1,109 @@
+
+## inspect-container — Inspect Container
+
+### SYNOPSIS
+
+`machinery inspect-container` [OPTIONS] IMAGENAME
+
+`machinery inspect-container` [OPTIONS] IMAGEID
+
+`machinery` help inspect-container
+
+
+### DESCRIPTION
+
+The `inspect-container` command analysis a container image by creating, starting and inspecting a container from the provided image
+and generates a system description from the gathered data. After the inspection the container will be killed and removed again.
+This approach ensures that no containers and images are affected by the inspection.
+
+Right now we support only images from the type `docker`, which is set as default.
```

Comment:
`, which is set as default.` Set as default for what? Can we skip this part of the sentence?